cli/src/lib.rs

```diff
@@ -179,89 +35,32 @@ fn process_to_kml(
-            // We set possible values on the arg value parser, we should not be able to reach here
-            unreachable!()
-        }
-    } else {
-        // Check extensions
-        formats
-            .values()
-            .filter(|Format { file_ext, .. }| Some(file_ext) == input_path.extension().as_ref())
-            .collect()
-    };
-
-    if formats.is_empty() {
+    if input_path.extension().as_ref() != Some(&OsStr::new("json")) {
         return Err(anyhow::anyhow!(
             "Could not load input file.\n\
-    Cannot determine format from extension; try using `--hint`"
+    Invalid file extension; the file should be json."
         ));
     }
 
-    let mut errors = Vec::with_capacity(formats.len());
-
-    for Format { initializer, .. } in formats.iter() {
-        match initializer.init_from_cli(cmd, args).and_then(|mut parser| {
-            let file = File::open(input_path)?;
-            parser.parse_file(&mut BufReader::new(file))
-        }) {
-            Ok(database) => {
-                return Ok(database);
-            }
-            Err(err) => errors.push(err),
+    let file = File::open(input_path)?;
+    let reader = &mut BufReader::new(file);
+    let parser: Result<Document, anyhow::Error> =
+        serde_json::from_reader(reader).map_err(anyhow::Error::from);
+    match parser {
+        Ok(mut database) => {
+            if let Some(origin) = args.get_one::<Position3d>("detection_origin") {
+                database.static_cuas_location = *origin;
+            };
+            return Ok(database);
         }
-    }
-
-    if formats.len() > 1 {
-        let error_str = {
-            formats
-                .iter()
-                .zip(errors.into_iter())
-                .map(|(format, error)| format!("\t- {} (Error: {})\n", format.name, error))
-                .collect::<String>()
-        };
-
-        Err(anyhow::anyhow!(
+        Err(err) => Err(anyhow::anyhow!(
             "Could not load input file.\n\
-    Tried with the following formats:\n\
-    {error_str}"
-        ))
-    } else {
-        Err(anyhow::anyhow!(
-            "Could not load input file.\n\
-    Tried loading it as a {} file, but got the following error: {}",
-            formats[0].name,
-            errors[0]
-        ))
-    }
-}
-
-pub struct CourageousParser;
-
-impl FormatParser for CourageousParser {
-    fn parse_file(&mut self, file: &mut BufReader<File>) -> Result<Document, anyhow::Error> {
-        serde_json::from_reader(file).map_err(anyhow::Error::from)
-    }
-}
-
-pub struct CourageousInitializer;
-
-impl CliFormatInitializer for CourageousInitializer {
-    fn init_from_cli(
-        &self,
-        _cmd: &Command,
-        _args: &ArgMatches,
-    ) -> Result<Box<dyn FormatParser>, anyhow::Error> {
-        Ok(Box::new(CourageousParser))
+        Tried loading it as a {} file, but got the following error: {}",
+            "COURAGEOUS (v0.4)", // COURAGEOUS (v0.4) should not be modified manually
```

Comment:
This comment could mean that 1) This value shouldn't be modified, or 2) This value should be modified, but we should try to find some way to do it automatically. Please clarify that. Also you can prefix the comment with `TODO: ` just to be able to find it later more easily.